wp-content/themes/access/functions.php

```diff
@@ -23,3 +24,20 @@
 require_once Path\controller('site');
 
 new Controller\Site();
+
+/**
+ * Adds functionality to Twig.
+ *
+ * @param \Twig\Environment $twig The Twig environment.
+ * @return \Twig\Environment
+ */
+function add_to_twig( $twig ) {
+    // Adding functions as filters.
+    $twig->addFilter( new Timber\Twig_Filter( 'url_decode', 'url_decode' ) );
+
+    return $twig;
+}
+
+function url_decode( $text ) {
+    return  urldecode($text);
+}
```

Comment:
@theasteve can you drop this block `add_to_twig` and `url_decode` in the lib/functions.php file?